okhttp-client/src/main/scala/org/http4s/client/okhttp/OkHttpBuilder.scala

```diff
@@ -166,7 +170,7 @@ sealed abstract class OkHttpBuilder[F[_]] private (
 
 /** Builder for a [[org.http4s.client.Client]] with an OkHttp backend
   *
-  * @define BLOCKINGEC an execution context onto which all blocking
+  * @define BLOCKER a [[cats.effect.Blocker]] onto which all blocking
```

Comment:
These external references usually fail at doc time.  I'd be delighted if that's no longer true.